test/testhelpers.sh

```diff
@@ -148,3 +148,9 @@ shutdown() {
     fi
   fi
 }
+
+assert_file_line_count() {
+  local filename="$1"
+  local lines="$2"
+  [ $(wc -l < "$filename") -eq "$lines" ]
```

Comment:
It doesn't save any lines, but I think it makes this assert easier to read and use. Anyways, I want this PR merged therefore I will revert this change :weary: